cross-env NODE_ENV=development && node ./api/api.js -- npm ERR! code ELIFECYCLE npm ERR! errno 1

F:\nodejs\Gold\express-graphql-boilerplate-master>npm start

> express-rest-api-boilerplate@0.0.0 prestart F:\nodejs\Gold\express-graphql-boilerplate-master
> npm run create-sqlite-db


> express-rest-api-boilerplate@0.0.0 create-sqlite-db F:\nodejs\Gold\express-graphql-boilerplate-master
> shx touch ./db/database.sqlite


> express-rest-api-boilerplate@0.0.0 start F:\nodejs\Gold\express-graphql-boilerplate-master
> npm run nodemon


> express-rest-api-boilerplate@0.0.0 nodemon F:\nodejs\Gold\express-graphql-boilerplate-master
> nodemon --exec npm run dev

[nodemon] 1.11.0
[nodemon] to restart at any time, enter `rs`
[nodemon] watching: *.*
[nodemon] starting `npm run dev`

> express-rest-api-boilerplate@0.0.0 dev F:\nodejs\Gold\express-graphql-boilerplate-master
> cross-env NODE_ENV=development && node ./api/api.js

only development, staging, test and production are valid NODE_ENV variables but undefined is specified
npm ERR! code ELIFECYCLE
npm ERR! errno 1
npm ERR! express-rest-api-boilerplate@0.0.0 dev: `cross-env NODE_ENV=development && node ./api/api.js`
npm ERR! Exit status 1
npm ERR!
npm ERR! Failed at the express-rest-api-boilerplate@0.0.0 dev script.
npm ERR! This is probably not a problem with npm. There is likely additional logging output above.

npm ERR! A complete log of this run can be found in:
npm ERR!     C:\Users\Sankar\AppData\Roaming\npm-cache\_logs\2017-12-25T03_40_52_904Z-debug.log
[nodemon] app crashed - waiting for file changes before starting...

@aichbauer > set NODE_ENV=development && node ./api/api.js

only development, staging, test and production are valid NODE_ENV variables but development  is specified
npm ERR! code ELIFECYCLE
npm ERR! errno 1
npm ERR! express-rest-api-boilerplate@0.0.0 dev: `set NODE_ENV=development && node ./api/api.js`
npm ERR! Exit status 1
npm ERR!
npm ERR! Failed at the express-rest-api-boilerplate@0.0.0 dev script.
npm ERR! This is probably not a problem with npm. There is likely additional logging output above.

npm ERR! A complete log of this run can be found in:
npm ERR!     C:\Users\Sankar\AppData\Roaming\npm-cache\_logs\2018-01-03T05_00_37_856Z-debug.log
[nodemon] app crashed - waiting for file changes before starting...

@aichbauer > set NODE_ENV=development && node ./api/api.js

only development, staging, test and production are valid NODE_ENV variables but development  is specified
npm ERR! code ELIFECYCLE
npm ERR! errno 1
npm ERR! express-rest-api-boilerplate@0.0.0 dev: `set NODE_ENV=development && node ./api/api.js`
npm ERR! Exit status 1
npm ERR!
npm ERR! Failed at the express-rest-api-boilerplate@0.0.0 dev script.
npm ERR! This is probably not a problem with npm. There is likely additional logging output above.

npm ERR! A complete log of this run can be found in:
npm ERR!     C:\Users\Sankar\AppData\Roaming\npm-cache\_logs\2018-01-03T05_00_37_856Z-debug.log
[nodemon] app crashed - waiting for file changes before starting...

--- a/package.json
+++ b/package.json
@@ -12,7 +12,7 @@
     "url": "https://github.com/aichbauer/express-rest-api-boilerplate/issues"
   },
   "scripts": {
-    "dev": "cross-env NODE_ENV=development && node ./api/api.js",
+    "dev": "cross-env NODE_ENV=development node ./api/api.js",
     "create-sqlite-db": "shx touch ./db/database.sqlite",
     "drop-sqlite-db": "shx rm ./db/database.sqlite",
     "lint": "eslint ./api/. ./config/. ./test/.",
@@ -21,8 +21,8 @@
     "nodemon": "nodemon --exec npm run dev",
     "prepush": "npm test; npm run drop-sqlite-db; npm run create-sqlite-db",
     "pretest": "npm run lint",
-    "production": "cross-env NODE_ENV=production && node ./api/api.js",
-    "test": "cross-env NODE_ENV=testing && nyc ava",
+    "production": "cross-env NODE_ENV=production node ./api/api.js",
+    "test": "cross-env NODE_ENV=testing nyc ava",
     "test-ci": "nyc ava --no-color"
   },
   "babel": {

